File: java/src/arrays/EvenNumberOfDigits.java
package arrays;

public class EvenNumberOfDigits {
  public int findNumbers(int[] numbers) {
    int count = 0;
    for (int n : numbers) {
      if (Integer.toString(n).length() % 2 == 0) {
        count += 1;
      }
    }
    return count;
  }
}

File: rs/src/arrays/even_number_of_digits.rs
pub fn find_numbers(numbers: Vec<i32>) -> i32 {
    let mut count = 0;
    numbers.into_iter().for_each(|n| {
        if n.to_string().len() % 2 == 0 {
            count += 1;
        }
    });
    count
}
